source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -55,24 +91,39 @@ Controls Glossary
 
       Outputs of a :term:`system` are often represented using the variable :math:`\mathbf{y}`, a column vector with one entry per :term:`output` (or thing we can measure). For example, if our :term:`system` had states for velocity and acceleration but our sensor could only measure velocity, our, our :term:`output` vector would only include the :term:`system`\'s velocity.
 
+   phase portrait
+      A graph of a function's value and its :term:`derivative` as they change in time, given some initial starting conditions. They are useful for analyzing system behavior (stable/unstable operating points, limit cycles, etc.) given a certain set of parameters or starting conditions. See `phase portrait <https://en.wikipedia.org/wiki/Phase_portrait>`__ on wikipedia for more info.
+
+   PID
+      Proportional-Integral-Derivative - A feedback controller which calculates a :term:`control signal` from a weighted sum of the :term:`error`, the rate of change of the error, and an accumulated sum of previous errors. See `PID controller <https://en.wikipedia.org/wiki/PID_controller>`__. on wikipedia for more info.
+
    plant
       The :term:`system` or collection of actuators being controlled.
 
    process variable
       The term used to describe the output of a :term:`plant` in the context of PID control.
 
+   r-squared
+      A statistical measurement of how well an `OLS (Ordinary Least Squares) fit <https://en.wikipedia.org/wiki/Ordinary_least_squares>`__ predicts a given set of input data. It is not impacted by the units of measurement. A value of 0.0 is a horrible fit, and 1.0 is a perfect fit. See `Coefficient_of_determination <https://en.wikipedia.org/wiki/Coefficient_of_determination>`__ on wikipedia for more info.
```

Comment:
"1.0 is a perfect fit, and 0.0 is just guessing the average value of your data."

Reply:
Generally like the explanation. I might skip the negative values portion and leave it with the hedge word of "typically". Most FRC cases shouldn't end up with negative values.... or if they do they probably need to be asking more questions than just looking at the glossary definition anyway.